src/relayclient/RelayClient.ts

```diff
@@ -398,7 +398,7 @@ export class RelayClient {
         this.logger.info(`Audit call failed for relay at URL: ${auditor}. Failed audit calls: ${failedAuditorsCount}/${auditors.length}`)
       }
     }
-    if (auditors.length === failedAuditorsCount) {
+    if (auditors.length === failedAuditorsCount && failedAuditorsCount !== 0) {
```

Comment:
I think it is better to check `this.config.auditorsCount === 0` cause otherwise there will be no log indicating that, i.e. there was only 1 relay so we could not audit a transaction with any other relay, etc.

Reply:
If both are the same, there is no difference if I check either for nonzero.
I just wanted to remove "all zero auditors failed" log..

Reply:
well, it is not same. you are comparing actual auditors array. it is NOT GUARANTEED to be equal in size to `config.auditorsCount`

Reply:
the `getAuditors()` method returns at most `config.auditorsCount` elements
so this code only has to filter out the "all [zero] auditors failed" log